pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -53,12 +56,21 @@ def load_earth_magnetic_anomaly(
         a pixel-registered grid is returned unless only the
         gridline-registered grid is available.
 
-    mag4km : bool
-        Choose the data version to use. The default is ``False``, which is
-        observed at sea level over oceanic regions and has no data over land.
-        Set ``mag4km`` to ``True`` to use a version where all observations
-        are relative to an altitude of 4 km above the geoid and include data
-        over land.
+    data_source : str
+        Select the source of the magnetic anomaly data.
+
+        Available options:
+
+        - **emag** : EMAG2 Global Earth Magnetic Anomaly Model [Default
+          option]. Only includes data is observed from sea level over
+          oceanic regions. See :gmt-datasets:`earth-mag.html`.
+
+        - **emag4km** : Use a version  of EMAG2 where all observations
+          are relative to an altitude of 4 km above the geoid and include
+          data over land
+
+        - **wdmam** : World Digital Magnetic Anomaly Map (WDMAM).
+          See :gmt-datasets:`earth-wdmam.html`
```

Comment:
I prefer to the name `emag2` instead, because `emag2` is the official name of the dataset (https://doi.org/10.7289/V5H70CVX), and `emag4km` can be renamed to `emag2_4km` instead.